Automattic/wp-calypso
# Seller Experience: Adding intent to tracks events

#### Changes proposed in this Pull Request

* This PR adds the `intent` prop to the tracks events on the onboarding flow. 

#### Testing instructions

* Enable the debug by running the following code in the browser's console:
 ```
 localStorage.debug='calypso:analytics'
 ```
* When you navigate through the onboarding pages, you'll see that the events have the intent prop.
